modules/mtl/arrow-mtl-data/src/main/kotlin/arrow/mtl/Kleisli.kt

```diff
@@ -23,55 +24,43 @@ fun <F, D, A> KleisliOf<F, D, A>.run(d: D): Kind<F, A> = fix().run(d)
  * @property run the arrow from [D] to `Kind<F, A>`.
  */
 @higherkind
-class Kleisli<F, D, A>(val run: (D) -> Kind<F, A>) : KleisliOf<F, D, A>, KleisliKindedJ<F, D, A> {
+class Kleisli<F, D, A>private constructor(val run: (D) -> Kind<F, A>) : KleisliOf<F, D, A>, KleisliKindedJ<F, D, A> {
```

Comment:
Can we move this to `0.11.x` ?

Reply:
This was done to prevent ambiguity between this and the other `run` function that no longer needs the monad parameter. Even if we were to keep that we'd need a replacement that has a different name than `run`. `runKleisli`?

Reply:
I think this can be solved with either `@JvmName` or a dummy param `Unit = Unit`.

Is this change blocking for the fix? I haven't backtraced the changes yet, but I can look deeper into the code next week.

Reply:
That is a good idea. I'll readd the deleted and modified methods then and mark them as deprecated instead.